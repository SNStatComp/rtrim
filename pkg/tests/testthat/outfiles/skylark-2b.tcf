FILE outfiles/skylark.dat
TITLE  skylark-2b: model 3 with covariate
NTIMES 8
NCOVARS 2
LABELS
Habitat
Cov2
End
MISSING 999
WEIGHT Absent

model 3
covariates 1
weighting off
overdisp on
serialcor on
run

