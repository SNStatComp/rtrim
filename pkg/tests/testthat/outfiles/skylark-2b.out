 TRIM 3.61 :  TRend analysis and Indices for Monitoring data 
 STATISTICS NETHERLANDS 

 Date/Time: 6-9-2016 9:25:20

 Title :  skylark-2b:

 The following  5 variables have been read from file: 
 Z:\Users\patrick\Work\CBS\TRIM\wine\skylark.dat

 1. Site             number of values:    55
 2. Time             number of values:     8
 3. Count            missing =        999
 4. HABITAT          number of values:  2
 5. COV2             number of values:  4

 Number of sites without positive counts (removed)       0

 Number of observed zero counts           0
 Number of observed positive counts     202
 Total number of observed counts        202
 Number of missing counts               238
 Total number of counts                 440

 Total count                         2536.0

 Sites containing more than 10% of the total count 
   Site Number  Observed Total    %   
         3        431.0         17.0
        37        266.0         10.5
        40        624.0         24.6

 Time Point Averages 
   TimePoint  Observations  Average   Index  
       1              25       8.52    1.00
       2              20       8.10    0.95
       3              30      10.90    1.28
       4              30      11.27    1.32
       5              28      12.75    1.50
       6              29      14.66    1.72
       7              22      17.00    2.00
       8              18      18.89    2.22

 RESULTS FOR MODEL: Effects for Each Time Point
 ----------------------------------------------
   Effects of covariate(s)
    HABITAT
 

 ESTIMATION METHOD = Generalised Estimating Equations


  Total time used:  1.36 seconds

  Estimated Overdispersion     =  1.162
  Estimated Serial Correlation =  0.227

 GOODNESS OF FIT
  Chi-square                154.50, df    133, p 0.0979
  Likelihood Ratio          159.64, df    133, p 0.0575
  AIC (up to a constant)   -106.36

 WALD-TEST FOR SIGNIFICANCE OF COVARIATES 
 Covariate      Wald-Test  df   p  
  HABITAT          21.55    7  0.0030

 PARAMETER ESTIMATES 

                    Additive      std.err.   Multiplicative  std.err.
 Constant
 --------
  Time      1         0.0000       0.0000         1.0000      0.0000
            2        -0.2165       0.1991         0.8053      0.1604
            3        -0.3781       0.2303         0.6851      0.1578
            4        -0.4982       0.2437         0.6076      0.1481
            5        -0.7392       0.2565         0.4775      0.1225
            6        -0.5213       0.2409         0.5938      0.1430
            7        -0.6366       0.2634         0.5291      0.1393
            8        -0.7215       0.2652         0.4860      0.1289

 Covariate  1
 ------------
 Category   2
  Time      1         0.0000       0.0000         1.0000      0.0000
            2        -0.1445       0.2324         0.8655      0.2011
            3         0.2771       0.2553         1.3194      0.3369
            4         0.3865       0.2681         1.4718      0.3946
            5         0.7747       0.2790         2.1700      0.6053
            6         0.6449       0.2643         1.9058      0.5036
            7         0.8588       0.2856         2.3603      0.6742
            8         1.0308       0.2883         2.8032      0.8081


 Time INDICES
  Time     Model     std.err.     Imputed    std.err.
     1     1                       1
     2     0.7281     0.0751       0.7234     0.0751
     3     0.8411     0.0846       0.8422     0.0848
     4     0.8119     0.0835       0.8145     0.0838
     5     0.8757     0.0886       0.8765     0.0888
     6     0.9771     0.0987       0.9792     0.0990
     7     1.0420     0.1068       1.0433     0.1071
     8     1.1106     0.1155       1.1219     0.1164

 TIME TOTALS
  Time      Model    std.err.     Imputed    std.err.
     1        526          44          526          44
     2        383          36          380          36
     3        443          30          443          30
     4        427          28          428          28
     5        461          29          461          29
     6        514          29          515          29
     7        548          34          548          34
     8        585          38          590          38

 OVERALL SLOPE MODEL: Moderate increase (p<0.01) **
  Additive      std.err.   Multiplicative  std.err.
    0.0363       0.0134         1.0370      0.0139

 OVERALL SLOPE IMPUTED+(recommended): Moderate increase (p<0.01) **
  Additive      std.err.   Multiplicative  std.err.
    0.0376       0.0134         1.0384      0.0140

